frontend/src/Styles/RolesAndRules.module.css

```diff
@@ -1,38 +1,100 @@
-@import './colourScheme.css';
-
-.roleContainer {
-    display: grid;
-    grid-template-columns: 1fr 1fr 1fr 1fr 1fr;
-    grid-template-rows: auto;
-    align-items: stretch;
-    justify-items: stretch;
-    text-align: center;
-    width: 100%;
-    column-gap: 3em;
-}
-
-.roleContainer h3 {
-    margin-top: 0;
-    text-transform: uppercase;
-}
-
-.ruleContainer {
-    margin: 3em auto 2em auto;
-    width: 80%;
-    text-align: justify;
-    white-space: pre-line;
-}
-
-.roleInfo {
-    display: grid;
-    grid-template-rows: 2em 1fr;
-}
-
-.roleHighlight {
-    background-color: var(--darkOrange);
-    color: black;
-    margin: 0;
-    padding: 0.5em;
-    border-radius: 10px 10px 0 0;
-    font-weight: 500;
-}
+@import './colourScheme.css';
+@import url('https://fonts.googleapis.com/css?family=Audiowide');
+
+.pageContainer {
+    align-items: center;
+    background-color: #A4B8C4!important;

```

Comment:
Nit: This is already defined in the `colorScheme.css` file as cadetBlueCrayola, and you used this in `ModulMUI.module.css`, but easy enough to fix up later.